src/index.html

```diff
@@ -11,12 +11,116 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500&
+      display=swap"
+      rel="stylesheet"
+    />
```

Comment:
The attributes of the <link> tag should be on new lines with proper indentation. Each attribute should start on a new line with a 2-space indentation from the tag.